reset.css

```diff
@@ -0,0 +1,53 @@
+*{
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+:root{
+    font-family: serif, sans-serif, Verdana, Tahoma;
+}
+html,body{
+    /* color-variables */
+    --bg-color:#7b7c7d6d;
+}
+a{
+    color: inherit;
+    text-decoration: inherit;
+    outline: none;
+    cursor: pointer;
+}
+.txt-space{
+    letter-spacing: 2px;
+}
+.pad-1{
+    padding: 1rem;
+}
+.pad-rht{
+    padding-right: 1rem;
+}
+.pad-lft{
+    padding-left: 1rem;
+}
+.w-full{
+    width: 100%;
+}
+.w-fit{
+    width: fit-content;
+}
+.box-shadow{
+    box-shadow: 0 0 3px 2px var(--secondary);
+}
+.border-btm{
+    border-bottom: 1px solid;
+}
+.d-column-centre{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.uppercase{
+    text-transform: uppercase;
+}
+.bolder{
+    font-weight: bolder;
+}
```

Comment:
Try to not use reset.css in as much as possible in your projects. You can use it from the one you set up in your component library. Use it only if you need somethings to be overwritten

Reply:
i will move those reset.css files in component library once i am done with this project